monitoring/grafana/dashboards/tests/features/ceph-cluster.feature

```diff
@@ -2,9 +2,53 @@ Feature: Ceph Cluster Dashboard
 
 Scenario: "Test total PG States"
   Given the following series:
-    | metrics | values |
+    | metrics                  | values |
     | ceph_pg_total{foo="var"} | 10 100 |
     | ceph_pg_total{foo="bar"} | 20 200 |
   Then Grafana panel `PG States` with legend `Total` shows:
     | metrics | values |
-    | {} | 300 |
\ No newline at end of file
+    | {}      | 300    |
+
+Scenario: "Test OSDs in"
+  Given the following series:
+    | metrics                          | values |
+    | ceph_osd_in{ceph_daemon="osd.0"} | 1.0    |
+    | ceph_osd_in{ceph_daemon="osd.1"} | 0.0    |
+    | ceph_osd_in{ceph_daemon="osd.2"} | 1.0    |
+  When variable `instance` is `.*`
+  Then Grafana panel `OSDs` with legend `In` shows:
+    | metrics | values |
+    | {}      | 3      |
```

Comment:
```suggestion
    | {}      | 2      |
```

Reply:
@p-se It seems this suggestion fixes the error that appears in `make check`:
```
  Scenario: "Test OSDs in"                           # tests/features/ceph-cluster.feature:12
    Given the following series                       # tests/features/environment.py:66
      | metrics                          | values |
      | ceph_osd_in{ceph_daemon="osd.0"} | 1.0    |
      | ceph_osd_in{ceph_daemon="osd.1"} | 0.0    |
      | ceph_osd_in{ceph_daemon="osd.2"} | 1.0    |
    When variable `instance` is `.*`                 # tests/features/environment.py:101
    Then Grafana panel `OSDs` with legend `In` shows # tests/features/environment.py:106
      | metrics | values |
      | {}      | 3      |
Unit Testing:  /tmp/tmpglyt66n3
  FAILED:
    expr: "sum(ceph_osd_in)", time: 1m0s,
        exp:"{} 3E+00"
        got:"{} 2E+00"

evaluation_interval: 1m
tests:
- input_series:
  - series: ceph_osd_in{ceph_daemon="osd.0"}
    values: '1.0'
  - series: ceph_osd_in{ceph_daemon="osd.1"}
    values: '0.0'
  - series: ceph_osd_in{ceph_daemon="osd.2"}
    values: '1.0'
  interval: 1m
  promql_expr_test:
  - eval_time: 1m
    exp_samples:
    - labels: '{}'
      value: 3.0
    expr: sum(ceph_osd_in)

None
HOOK-ERROR in after_scenario: AssertionError:
```